python/semantic_kernel/connectors/ai/chat_completion_client_base.py

```diff
@@ -303,8 +306,18 @@ async def get_streaming_chat_message_contents(
                     ],
                 )
 
+                # Merge and yield the function results, regardless of the termination status
+                # Include the ai_model_id so we can later add two streaming messages together
+                # Some settings may not have an ai_model_id, so we need to check for it
+                ai_model_id = self._get_ai_model_id(settings)
```

Comment:
The base class `AIServiceClientBase` also has a property `ai_model_id`, should we check there too?

Reply:
Definitely, thanks for the callout. Updated to fallback to the `self.ai_model_id` if `settings.ai_model_id` doesn't exist (or is blank). (Side note: it's interesting that our `ai_model_id` in the prompt execution settings allows or "" strings.)

Reply:
I don't see the update. Am I missing something?

Reply:
I haven't pushed it yet. Will push now.